build.gradle

```diff
@@ -110,7 +91,7 @@ if (!usingRemoteCluster) {
 }
 
 ext {
-    opendistroVersion = '1.15.0'
+    opendistroVersion = '1.0.0'
```

Comment:
Why not update opendistroversion here to opensearch?

Reply:
Will target complete renaming variable/namespace in next phase.